src/components/TeamCard/TeamCard.scss

```diff
@@ -6,8 +6,10 @@
   max-width: 292px;
   min-height: 392px;
   max-height: 500px;
-  margin-left: 10px;
-  margin-right: 10px;
+  margin: 3em 2.5em 0;
+  @media (max-width: 320px) {
+    margin: 3em 10px 0;
+  }
   border-radius: 27px;
   padding-bottom: 10px;
```

Comment:
We really want to avoid mixing padding and margin on the same component because we can get some weird effects